client/cloudbuild.yaml

```diff
@@ -73,5 +74,7 @@ availableSecrets:
     env: 'REGION'
   - versionName: projects/${PROJECT_ID}/secrets/ROOT_URL/versions/${_ENV}
     env: 'ROOT_URL'
+  - versionName: projects/${PROJECT_ID}/secrets/WEBSOCKET_URL/versions/${_ENV}
+    env: 'WEBSOCKET_URL'
```

Comment:
It doesn't look like this has been added in[ secret manager](https://console.cloud.google.com/security/secret-manager?referrer=search&project=wondertix-app) yet. Do you know what values we would want to use for dev, stg, and prod?

Reply:
The general format of the websocket URL should look like:

`wss://{root_location}/wss`

So if the root URL is `https://rooturl.com`, the Websocket URL should be `wss://rooturl.com/wss`. This should apply for the dev, stg, and prod.

It is worth noting that the `/wss` at the end is not strictly necessary, it can be anything or nothing. It just seemed to be a reasonable name, and keeping it consistent allows us to easily use the [ws library feature](https://github.com/websockets/ws/blob/HEAD/doc/ws.md#new-websocketserveroptions-callback) to limit connections to a specific path in the future if we need to for whatever reason.

Reply:
@ben16williams Once we have a secret in Secret Manager, this looks good to me

Reply:
Done https://console.cloud.google.com/security/secret-manager/secret/WEBSOCKET_URL/versions?project=wondertix-app. dev is configured as `wss://dev.wondertix.com/wss`, staging is `wss://stg.wondertix.com/wss`, and production is `wss://www.wondertix.com/wss`